src/core/environment.js

```diff
@@ -246,12 +246,18 @@ p5.prototype.cursor = function(type, x, y) {
  * frames per second (usual for movies) or above will be enough for smooth
  * animations. This is the same as setFrameRate(val).
  *
- * Calling <a href="#/p5/frameRate">frameRate()</a> with no arguments returns
- * the current framerate. The draw function must run at least once before it will
- * return a value. This is the same as <a href="#/p5/getFrameRate">getFrameRate()</a>.
- *
- * Calling <a href="#/p5/frameRate">frameRate()</a> with arguments that are not
- * of the type Number or are non-positive also returns current framerate.
+ * Calling <a href="#/p5/frameRate">frameRate()</a> with no arguments or
+ * with arguments that are not of type Number or are non-positive returns
+ * an approximation of the current frame rate. The draw function must run at
+ * least once before it will return a value. This is the same as using
+ * <a href="#/p5/getFrameRate">getFrameRate()</a>.
```

Comment:
Just noticed that this link is dead (possibly because this method is private?) https://github.com/processing/p5.js/blob/8e117ae3dd1ac44f8e42bcd1d350a4cde6c36b05/src/core/environment.js#L318-L321

We can decide later to take this out if we want, but for now maybe we should at least take out the link (similar to how `setFrameRate` is documented on line 247.)